sdk/src/wallet/core/operations/address_generation.rs

```diff
@@ -36,41 +36,34 @@ impl Wallet {
                 // needs to have it visible on the computer first, so we need to generate it without the
                 // prompt first
                 let options = options.into();
+                #[cfg(feature = "events")]
```

Comment:
So this emits before generating now?

Reply:
it always did. https://github.com/iotaledger/iota-sdk/blob/develop/sdk/src/wallet/core/operations/address_generation.rs#L62

Reply:
Ah, diffs are hard 😆